### PR TITLE
Changed object-models to create schema IR-1159

### PR DESCRIPTION
**Pull request by rkharisov**
Signed-off-by: rkharisov <rinat@soramitsu.co.jp>

### Jira Issue
https://jira.hyperledger.org/browse/IR-1159

### CODE REVIEW

**Review by eadventurous — DISMISSED** [review]

File: iroha_schema/iroha_schema_bin/src/main.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 use std::collections::BTreeMap;
 
-use iroha_data_model::{prelude::*, query::QueryBox};
+use iroha_data_model::{prelude::*};
```
Comment (i1i1) [inline iroha_schema/iroha_schema_bin/src/main.rs:7]: Styling!

**Review by i1i1 — CHANGES_REQUESTED** [review]

File: iroha_schema/iroha_schema_bin/src/main.rs (hunk 1 above)
Comment (eadventurous) [reply iroha_schema/iroha_schema_bin/src/main.rs:7]: Yes, the formatter caught it

Comment (i1i1) [reply iroha_schema/iroha_schema_bin/src/main.rs:7]: Made a comment to make sure that it wont be merged till it is fixed :)

**Review by eadventurous — APPROVED** [review]

**Review by i1i1 — APPROVED** [review]

File: iroha_schema/iroha_schema_bin/src/main.rs (hunk 1 above)
Comment (rkharisov) [reply iroha_schema/iroha_schema_bin/src/main.rs]: fixed

**Pull request merged**